src/reducers/likedVideosReducer.jsx

```diff
@@ -0,0 +1,14 @@
+function LikedVideosReducer(state, action){
+    
+    switch(action.type){
+        case 'LIKE_TOGGLE':{
+            if (state.filter(item => item._id==action.payload._id).length <= 0){

```

Comment:
I would suggest `.some()` array method for this particular case just so that you don't have to `filter` and check its `length`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
This looks interesting. Coming across this for the first time. Thanks Vinay. Will try implementing it.